### PR TITLE
스페이스 관심/관심 해제

### PR DESCRIPTION
**Pull request by Seohyun-Roh**

### CODE REVIEW

**Comment by Seohyun-Roh** [comment]
Current dependencies on/for this PR:
* main
  * **PR #234** <a href="https://app.graphite.dev/github/pr/penxle/penxle/234" target="_blank">Graphite</a> 
    * **PR #240** <a href="https://app.graphite.dev/github/pr/penxle/penxle/240" target="_blank">Graphite</a> 
      * **PR #246** <a href="https://app.graphite.dev/github/pr/penxle/penxle/246" target="_blank">Graphite</a>  👈
    * **PR #235** <a href="https://app.graphite.dev/github/pr/penxle/penxle/235" target="_blank">Graphite</a> 
      * **PR #238** <a href="https://app.graphite.dev/github/pr/penxle/penxle/238" target="_blank">Graphite</a> 

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/penxle/penxle/246?utm_source=stack-comment).

File: apps/penxle.com/src/routes/(default)/[space]/+layout@.svelte [hunk 1]
```diff
@@ -122,3 +225,13 @@
...
+
+<Modal size="sm" bind:open={moveToLoginOpen}>
+  <svelte:fragment slot="title">로그인이 필요한 기능이에요</svelte:fragment>
+  <svelte:fragment slot="subtitle">로그인 하고 펜슬의 더 다양한 기능을 사용해 보세요!</svelte:fragment>
+
+  <div slot="action" class="flex w-full gap-3">
+    <Button class="w-full" color="secondary" size="xl" on:click={() => (moveToLoginOpen = false)}>닫기</Button>
+    <Button class="w-full" href="/login" size="xl" type="link">로그인</Button>
+  </div>
+</Modal>
```
Comment (devunt) [inline apps/penxle.com/src/routes/(default)/[space]/+layout@.svelte:237]: P3: 요거 로그인하러 가기 모달은 UI 곳곳에서 쓰일 거 같은데 따로 컴포넌트로 빼는거 어떠신가용

File: apps/penxle.com/src/routes/(default)/[space]/+layout@.svelte [hunk 2]
```diff
@@ -22,10 +35,34 @@
...
+        id
+
+        followingSpaces {
+          id
+        }
+      }
+    }
+  `);
+
+  $: following = $query.me?.followingSpaces.find((space) => space.id === $query.space.id);
```
Comment (devunt) [inline apps/penxle.com/src/routes/(default)/[space]/+layout@.svelte:65]: P2: 요거 팔로우하고 있는지 여부 프론트에서 계산하지 말고 GQL로 따로 빼는게 좋을 듯 (서버 따로 작업해둘게요)

Comment (devunt) [reply apps/penxle.com/src/routes/(default)/[space]/+layout@.svelte:65]: `feature/user-space-followed` 에 작업해두었습니다

**Review by devunt — APPROVED** [review]

**Comment by devunt** [comment]
### Merge activity

* **Oct 25, 7:34 AM**: @devunt started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/penxle/penxle/246).

**Pull request merged**